src/CMakeLists.txt

```diff
@@ -29,36 +29,39 @@ cmake_policy(SET CMP0057 NEW)
 include (GenerateExportHeader)
 add_subdirectory(sqlite)
 
-# Truncation rounding or (default) rounding to nearest even (RNE) is enabled.
-# This switch controls two related but different aspects of MIOpen behavior
-# 1.  How host code performs conversions of float to bfloat16, important only
-#     for testing.
-# 2.  How BF16 kernels (which are kind of mixed-precision now and expected to
-#     remain in the future)  perform final conversion (and rounding) of FP32
-#     to BF16 results. This affects the main functionality of the library.
+#Truncation rounding or (default) rounding to nearest even(RNE) is enabled.
+#This switch controls two related but different aspects of MIOpen behavior
+# 1. How host code performs conversions of float to bfloat16, important only
+#for testing.
+# 2. How BF16 kernels(which are kind of mixed - precision now and expected to
+#remain in the future) perform final conversion(and rounding) of FP32
+#to BF16 results.This affects the main functionality of the library.
 option( MIOPEN_USE_RNE_BFLOAT16 "Sets rounding scheme for bfloat16 type" ON )
 
 configure_file("${PROJECT_SOURCE_DIR}/include/miopen/config.h.in" "${PROJECT_BINARY_DIR}/include/miopen/config.h")
 
-# configure a header file to pass the CMake version settings to the source, and package the header files in the output archive
+#configure a header file to pass the CMake version settings to the source, \
+    and package the header files in the output archive
 configure_file( "${PROJECT_SOURCE_DIR}/include/miopen/version.h.in" "${PROJECT_BINARY_DIR}/include/miopen/version.h" )
 
 message( STATUS "MIOpen_VERSION= ${MIOpen_VERSION}" )
 message( STATUS "CMAKE_BUILD_TYPE= ${CMAKE_BUILD_TYPE}" )
 
-# This is incremented when the ABI to the library changes
+#This is incremented when the ABI to the library changes
 set( MIOpen_SOVERSION 1.0 )
 
 
 function(add_kernels KERNEL_FILES)
     set(INIT_KERNELS_LIST)
     foreach(KERNEL_FILE ${KERNEL_FILES})
         if("${CMAKE_VERSION}" VERSION_LESS 3.0)
-            configure_file(${KERNEL_FILE} ${KERNEL_FILE}.delete)
+            configure_file(${
```

Comment:
Looks like you formatted your CMakeLists.txt
Please revert the formatting to all CMake files as it can introduce bugs.

Reply:
Sure Daniel.